File: version-1/single-number.cpp
//算法复杂度不满足leetcode的要求，待改进
int singleNumber(int* nums, int numsSize) {
    int result,i,j;
    bool flag=true;
    for(i=0;i<numsSize;i++){
       flag=true;
       for(j=i+1;j<numsSize;j++){
	   if(nums[i]==nums[j]){
	   	flag=false;
	   	continue;
	   }
	   }
	   if(flag){
	   	result=nums[i];
	   	break;
	   }
	}
	return result;
}

File: version-1/single-number.py
#single number
#先排序，再找就简单了
def singleNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        flag=True
        nums.sort()
        for i in range(len(nums)-1)[::2]:
            if nums[i]!=nums[i+1]:
              flag=False
              return nums[i]
              break
        if  flag:
          return nums[len(nums)-1]